setup.py

```diff
@@ -1 +1,54 @@
-generated by ci
+from setuptools import setup
+import datetime
+import unittest
+import os
+
+def get_version():
+    _version_file = 'taskcat/.version'
+
+    if os.path.exists(_version_file) and os.path.getsize(_version_file) > 0:
+        _version = open(_version_file, 'r').read()
+    else:
```

Comment:
Would prefer this fails if `taskcat/.version` isn't present. Otherwise the bug outlined in #129 lives on, and will cause longer term issues.

Reply:
What if we mod setup.py to only do a version bump when _build_ is passed in as an argument to setup.py. Then update this [line in travis.yml](https://github.com/aws-quickstart/taskcat/blob/develop/.travis.yml#L9) to `python setup.py build && python setup.py install`. ?

Reply:
This will address the dynamic version bump. It doesn't address the need to bail if taskcat/.version doesn't exist, and we're not passing build. Should we bail?